DIPs/DIP.md

```diff
@@ -57,6 +57,21 @@ On the contrary the following must not compile:
     auto t = T(a:4);
     auto t = T("test", a:4);
 
+As this is about struct initialization it isn't influenced in any way to
+constructors. In consequence we have the following:
+
+  struct U {
+    int i;
```

Comment:
BTW the Markdown formatting is broken - only indented lines get a fixed-space font. Tip: use \`\`\`d on its own line before the struct line and \`\`\` after the assert lines. (Make sure there's still blank lines before and after the block or MD gets confused).